tests/cases/fourslash/findAllRefsJsDocTypeDef_js.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path='fourslash.ts'/>
+
+// Tests that the scope of @typedef is not just the node immediately below it.
+
+// @allowJs: true
+
+// @Filename: /a.js
+/////** @typedef {number} [|{| "isWriteAccess": true, "isDefinition": true |}T|] */
+////
+/////**
+//// * @return {[|T|]}
+//// */
+////function f(obj) { return 0; }
+////
+/////**
+//// * @return {[|T|]}
+//// */
+////function f2(obj) { return 0; }
+
+const [r0, r1, r2] = test.ranges();
+verify.singleReferenceGroup("type T = number");
```

Comment:
I haven't used `singleReferenceGroup`; does it interact with the result of `test.ranges` somehow? I expected to see r0, r1, r2 passed in to it somehow.